Documentation/guides.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<guides xmlns="https://www.phpdoc.org/guides" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.phpdoc.org/guides ../vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
+        links-are-relative="true"
+>
+  <extension class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension"
+             project-home="https://extensions.typo3.org/extension/typo3_console"
+             project-contact="https://typo3.slack.com/archives/C07NARQGY"
+             project-repository="https://github.com/TYPO3-Console/TYPO3-Console"
+             project-issues="https://github.com/TYPO3-Console/TYPO3-Console/issues"
+             edit-on-github-branch="main"
+             edit-on-github="TYPO3-Console/TYPO3-Console"
+             interlink-shortcode="helhum/typo3-console"
+             typo3-core-preferred="stable"
+  />
+  <project title="TYPO3 Console" release="8.1.1" version="8.1.1"
```

Comment:
if this must contain the current version number, then this [method](https://github.com/TYPO3-Console/TYPO3-Console/blob/7916d393fe18b441d0e339fde6c9247fd49a5a58/Classes/Console/Composer/ScriptHelper.php#L30-L73), needs to be updated to change this number accordingly

Reply:
Nope this is only the version that is displayed when you try out the rendering locally. When it is rendered on the server It is automatically taken form the tag / composer version. Usually we put main here, will update that